x-pack/packages/kbn-elastic-assistant/impl/assistant/use_current_conversation/index.tsx

```diff
@@ -248,10 +258,19 @@ export const useCurrentConversation = ({
       });
       return;
     }
+    const newSystemPrompt = getDefaultNewSystemPrompt(allSystemPrompts);
 
     const newConversation = await createConversation({
       title: NEW_CHAT,
-      apiConfig: currentConversation?.apiConfig,
+      ...(currentConversation?.apiConfig != null &&
+      currentConversation?.apiConfig?.actionTypeId !== null
```

Comment:
Consider `currentConversation?.apiConfig?.actionTypeId != null` if this isn't testing for `null` specifically